modules/glpk/5.0.bcr.4/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "http://ftp.gnu.org/gnu/glpk/glpk-5.0.tar.gz",
```

Comment:
Please change to `https` and also for `repository` in metadata.json

Reply:
DONE / PTAL